ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/ontologies/OntologyDisplayStrings.properties

```diff
@@ -6,7 +6,7 @@ fileUploadMaxSizeWarning = Maximum size of each file is 1.9GB when using simple
 reset = Reset
 fileUploadFailed = The file {0} failed to upload.
 fileUploadSuccess = The file {0} was uploaded successfully.
-publishOntology = Publish this Ontology to the DE
+publishOntology = Publish
```

Comment:
Since this string doesn't say anything about ontologies anymore, I'd rename this property to just `publish`.
And it's now generic enough, maybe it can be moved to `de-lib/src/main/resources/org/iplantc/de/resources/client/messages/IplantDisplayStrings.properties`, if it's not too much trouble.